### PR TITLE
Included Database Driven Tutorials can you please merge?

### PR DESCRIPTION
**Pull request by Leo-G**

### CODE REVIEW

**Comment by humiaozuzu** [comment]
Sorry for the late reply.
New [contributing guide](https://github.com/humiaozuzu/awesome-flask/blob/master/CONTRIBUTING.md) are added. 

Please organize your posts as series and update the post structure and code snippets.

**Comment by Leo-G** [comment]
Please organize your posts as series and update the post structure and code snippets. I did not get posts as series??, Also my app https://github.com/Leo-G/Flask-Scaffold is currently trending but I am not sure why you need me to support python 2 especially if it is being deprecated.

**Comment by humiaozuzu** [comment]
@Leo-G  Organize as series means you add only one link here and users can find all your other series posts in that post. Just like [this one](http://blog.miguelgrinberg.com/post/the-flask-mega-tutorial-part-i-hello-world).

The code snippets in [this post](http://techarena51.com/index.php/deploy-flask-on-ubuntu/ ) is missing indent.

I did not mean you should support python 2 here, support is preferred.

**Comment by Leo-G** [comment]
Thank you  for clearing my doubts, I will make the corrections and get Back

**Comment by Leo-G** [comment]
Is it ok if I submit just this link http://techarena51.com/index.php/category/flask-framework-tutorials-and-examples/, It has a list

**Comment by humiaozuzu** [comment]
@Leo-G Yeah, it's OK

**Comment by Leo-G** [comment]
Done

File: README.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ Resources
     - [Discover Flask - Full Stack Web Development with Flask](http://discoverflask.com)
     - [Flaskr - Intro to Flask, Test Driven Development, and jQuery](https://github.com/mjhea0/flaskr-tdd)
     - [Real Python: Flask Category](https://realpython.com/blog/categories/flask/)
+    - [Flask-SQLAlchemy Tutorials and Examples](http://techarena51.com/index.php/category/flask-framework-tutorials-and-examples/)
+
```
Comment (humiaozuzu) [inline README.md]: @Leo-G Please remove the additional line here

**Comment by Leo-G** [comment]
done

**Pull request merged**